tests/core/notification/test_notifier.py

```diff
@@ -506,7 +506,7 @@ def test_publish_event():
         t.properties.update({"debug": True})
         assert registration_queue.qsize() == 0
 
-        d.name = "another new datanode name"
+        d.editor_id = "another new editor id"
```

Comment:
Why not test the name here? name is still exposed as a property, so it should still work.

Reply:
the changes attribute will be under `properties` and not `name`, but we have already tested changes to `properties` so it's not needed